backend/routes/connection.js

```diff
@@ -0,0 +1,51 @@
+// TODO: require postgres
+
+// TODO: if cloudsql disconnected
+
+// TODO: if cloudsql errors out
+
+// TODO: once connected, log "connect to cloudsql URI"
```

Comment:
I don't think so, since the knex client doesn't maintain an open connection with the database, but connects and closes the connection for each query. I turned debug mode on in the constructor, so that logs info with each client. As for connection errors, they're handled just like validation errors are- caught in `catch` blocks.